packages/aws-arch/scripts/common.ts

```diff
@@ -0,0 +1,29 @@
+import * as fs from 'fs-extra';
```

Comment:
Missing ASL header. Not sure why the header eslint plugin didn't add this automatically.